git_pw/patch.py

```diff
@@ -329,6 +329,24 @@ def update_cmd(patch_ids, commit_ref, state, delegate, archived, fmt):
     is_flag=True,
     help='Include patches that are archived.',
 )
+@click.option(
+    '--since',
+    'since',
+    metavar='SINCE',
+    help=(
+        'Show only patches since a given date. Should be in ISO 8601 '
+        'format, e.g. \'2023-05-09T00:00:00\''
+    ),
+)
+@click.option(
+    '--before',
+    'before',
+    metavar='BEFORE',
+    help=(
+        'Show only patches before a given date. Should be in ISO 8601 '
+        'format, e.g. \'2023-05-09T00:00:00\''
+    ),
+)
```

Comment:
To avoid some unnecessary duplication, I wonder could you move this to the `utils` module and create a new decorator like we do for pagination (see `pagination_options` below)?

If you do, you probably want to replace `patches` in the help text above with...say...`resources`? Whatever `pagination_options` uses.